gnssanalysis/gn_download.py

```diff
@@ -830,12 +831,15 @@ def download_product_from_cddis(
     logging.debug(
         f"Generated filename: {product_filename}, with GPS Date: {gps_date.gpswkD} and reference: {reference_start}"
     )
+
+    ensure_folders([download_dir])
+    download_filepaths = []
```

Comment:
This is not returned at the end (if that's what we want to change the behaviour to)